docs/core/tutorials/using-on-mac-vs-full-solution.md

```diff
@@ -184,7 +190,6 @@ Unit tests provide automated software testing during your development and publis
 
    ![The search word count is changed to a value of 999 in the app's output](./media/using-on-mac-vs-full-solution/vsmacfull19.png)
 
-## Next steps
+## See also
 
-* Explore additional features of Visual Studio for Mac in an [Introducing Visual Studio for Mac](https://developer.xamarin.com/guides/cross-platform/visual-studio-mac/) at the Xamarin Developer's website.
-* For a more in depth review of Visual Studio for Mac's features, refer to the [Xamarin Studio Tour](https://developer.xamarin.com/guides/cross-platform/xamarin-studio/ide-tour/) guide.
+[Visual Studio 2017 for Mac Release Notes](https://www.visualstudio.com/news/releasenotes/vs2017-mac-relnotes) at the Xamarin Developer's website.
```

Comment:
One more thing. Sorry! This is now at the Visual Studio site not at the Xamarin Developer's website.